packages/shorebird_cli/lib/src/executables/gradlew.dart

```diff
@@ -113,3 +117,16 @@ class Gradlew {
     return productFlavors;
   }
 }
+
+extension on String {
+  bool get isAcronymPrefixed {
```

Comment:
This felt like too specific to this command and to gradle names, so I decided to keep them private.

No strong opinions though, happy to change if others think different.